.env.example

```diff
@@ -0,0 +1,77 @@
+VITE_UNCHAINED_ARBITRUM_NOVA_HTTP_URL="https://api.arbitrum-nova.shapeshift.com"
+VITE_UNCHAINED_ARBITRUM_NOVA_WS_URL="wss://api.arbitrum-nova.shapeshift.com"
+VITE_UNCHAINED_BASE_HTTP_URL="https://api.base.shapeshift.com"
+VITE_UNCHAINED_BASE_WS_URL="wss://api.base.shapeshift.com"
+VITE_UNCHAINED_BITCOIN_HTTP_URL="https://api.bitcoin.shapeshift.com"
+VITE_UNCHAINED_BITCOIN_WS_URL="wss://api.bitcoin.shapeshift.com"
+VITE_UNCHAINED_DOGECOIN_HTTP_URL="https://api.dogecoin.shapeshift.com"
+VITE_UNCHAINED_DOGECOIN_WS_URL="wss://api.dogecoin.shapeshift.com"
+VITE_UNCHAINED_LITECOIN_HTTP_URL="https://api.litecoin.shapeshift.com"
+VITE_UNCHAINED_LITECOIN_WS_URL="wss://api.litecoin.shapeshift.com"
+VITE_UNCHAINED_BITCOINCASH_HTTP_URL="https://api.bitcoincash.shapeshift.com"
+VITE_UNCHAINED_BITCOINCASH_WS_URL="wss://api.bitcoincash.shapeshift.com"
+VITE_UNCHAINED_COSMOS_HTTP_URL="https://api.cosmos.shapeshift.com"
+VITE_UNCHAINED_COSMOS_WS_URL="wss://api.cosmos.shapeshift.com"
+VITE_UNCHAINED_THORCHAIN_HTTP_URL="https://api.thorchain.shapeshift.com"
+VITE_UNCHAINED_THORCHAIN_WS_URL="wss://api.thorchain.shapeshift.com"
+VITE_UNCHAINED_MAYACHAIN_HTTP_URL="https://api.mayachain.shapeshift.com"
+VITE_UNCHAINED_MAYACHAIN_WS_URL="wss://api.mayachain.shapeshift.com"
+VITE_UNCHAINED_SOLANA_HTTP_URL="https://api.solana.shapeshift.com"
+VITE_UNCHAINED_SOLANA_WS_URL="wss://api.solana.shapeshift.com"
+
+# Node URLs (Production)
+VITE_ETHEREUM_NODE_URL="https://api.ethereum.shapeshift.com/api/v1/jsonrpc"
+VITE_AVALANCHE_NODE_URL="https://api.avalanche.shapeshift.com/api/v1/jsonrpc"
+VITE_OPTIMISM_NODE_URL="https://api.optimism.shapeshift.com/api/v1/jsonrpc"
+VITE_BNBSMARTCHAIN_NODE_URL="https://api.bnbsmartchain.shapeshift.com/api/v1/jsonrpc"
+VITE_POLYGON_NODE_URL="https://api.polygon.shapeshift.com/api/v1/jsonrpc"
+VITE_GNOSIS_NODE_URL="https://daemon.gnosis.shapeshift.com"
+VITE_ARBITRUM_NODE_URL="https://api.arbitrum.shapeshift.com/api/v1/jsonrpc"
+VITE_ARBITRUM_NOVA_NODE_URL="https://api.arbitrum-nova.shapeshift.com/api/v1/jsonrpc"
+VITE_BASE_NODE_URL="https://api.base.shapeshift.com/api/v1/jsonrpc"
+VITE_THORCHAIN_NODE_URL="https://daemon.thorchain.shapeshift.com/lcd"
+VITE_MAYACHAIN_NODE_URL="https://daemon.mayachain.shapeshift.com/lcd"
+VITE_SOLANA_NODE_URL="https://api.solana.shapeshift.com/api/v1/jsonrpc"
+
+# Midgard URLs (Production)
+VITE_THORCHAIN_MIDGARD_URL="https://indexer.thorchain.shapeshift.com/v2"
+VITE_MAYACHAIN_MIDGARD_URL="https://indexer.mayachain.shapeshift.com/v2"
+
+# Swapper API URLs
+VITE_COWSWAP_BASE_URL="https://api.cow.fi"
+VITE_CHAINFLIP_API_KEY="09bc0796ff40435482c0a54fa6ae2784"
+VITE_CHAINFLIP_API_URL="https://chainflip-broker.io"
+VITE_RELAY_API_URL="https://api.relay.link"
+VITE_PORTALS_BASE_URL="https://api.portals.fi"
+VITE_ZRX_BASE_URL="https://api.proxy.shapeshift.com/api/v1/zrx/"
+
+# Additional APIs (used in swaps service)
+VITE_JUPITER_API_URL="https://quote-api.jup.ag/v6"
+
+SWAPS_SERVICE_URL=http://localhost:3001
```

Comment:
suggestion: probably fine for now and no need to change this in this PR but 3001 is used if you run two web instances, should probably use something else

Reply:
I'm happy to use anything else if needed but well having 2 web instance is an edge case more than the main case, if we launch web after the server it will probably use a free to use port no?